templates/includes/sidebar.html

```diff
@@ -1,4 +1,13 @@
 <ul class="sidebar-nav">
+  {% if user.is_authenticated %}
+    <li>User: {{ user.username }}</li>
+    <li><a href="{% url 'logout' %}">Logout</a></li>
+  {% else %}
+    <li><a href="{% url 'login' %}">Login</a></li>
+  {% endif %}
+
+  <br>
+
```

Comment:
Display the username on the top of the sidebar and login/logout button depending on if the user is authenticated. If the driver clicks on username - the corresponding Driver detail page must open.

Reply:
Oh, yeah, sure, I lost it